tests/admctrl_configmap_test.go

```diff
@@ -20,6 +21,8 @@ import (
 )
 
 func TestAdmissionControllerConfigMap(t *testing.T) {
+	t.Setenv(features.NewPolicyCategories.Name(), "true")
```

Comment:
Did you add this test through your mammoth feature PR? If not, why cannot we run this test anymore with feature flag off?

Reply:
Wait both the postgres and non-postgres will run the same file. So does that mean the one that was working fine will now get the flag and thus might cause other issues?

Reply:
Let's wait for CI, but I don't think so.

Reply:
Is the current fix enough on release builds e.g. nightly?

Reply:
Is there a way to find out until a nightly runs with it? Or is there a tag to run the nightlies on this PR?

Also looks like some infra blip causing the e2e tests to fail

Reply:
Alright, CI has other plans. So the wait and watch is longer. 🤷‍♀️